app/models/ncr/work_order.rb

```diff
@@ -57,6 +60,10 @@ def self.purchase_amount_column_name
     }
 
     def set_defaults
+      # not sure why the latter condition is necessary...was getting some weird errors from the tests without it. -AF 10/5/2015
```

Comment:
can we try to get to the bottom of this instead of leaving a comment? Or at least file a tracker story about it?